README.md

```diff
@@ -245,6 +245,7 @@ includes inlining of methods within the scala library (which avoids lambda alloc
 only safe if the Scala library version is the same at compile time and runtime. Since we cannot guarantee this for all cases (for example, users
 may depend on the kafka jar for integration tests where they may include a scala library with a different version), we don't enable it by
 default. See https://www.lightbend.com/blog/scala-inliner-optimizer for more details.
+* `checkstyleVersion`: override CheckStyle dependency version
```

Comment:
question: do you think we should add some note in the above `Checkstyle` section?

Reply:
Sure @showuon ! Let me provide some more explanation here.

Reply:
Ok, I just added few more details and force-pushed. 

I am going to resolve this conversation, but feel free to open a new one @showuon in case you want to suggest anything else).